app/assets/javascripts/monotile/hat.js

```diff
@@ -0,0 +1,466 @@
+// BSD-3-Clause licensed by Craig S. Kaplan
+// adapted from: https://github.com/isohedral/hatviz
+
+// A bit of a "hacky" fix. This is the only way I found that works
+// so that the canvas is loaded even if the site is left
+// e.g. user clicks on "Register" or changes language from "De" to "En"
+// with the button on the landing page.
+// Also see the following link for a possible workaround, which does not work
+// for us in conjunction with turbolink: 
+// https://github.com/processing/p5.js/wiki/p5.js-overview#why-cant-i-assign-variables-using-p5-functions-and-variables-before-setup
+$(document).on('turbolinks:load', () => {
+	try {
+		setup();
+	} catch (err) {
+		if (err instanceof ReferenceError) {
+			return;
+		}
+		console.error(err);
+	}
+});
+
+const INITIAL_TO_SCREEN = [45, 0, 0, 0, -45, 0];
+let to_screen = INITIAL_TO_SCREEN;
+let tiles;
+let level;
+let box_height;
+
+let monotile_btn;
+let reset_btn;
+
+const colors = {
+	'H1': [0, 137, 212],
+	'H': [148, 205, 235],
+	'T': [251, 251, 251],
+	'P': [250, 250, 250],
+	'F': [191, 191, 191]
+};
```

Comment:
I have shown the new landing screen to several people. There were some complaints about the dark grey. I experimented myself a little with the colors and arrived at these values:
```javascript
const colors = {
	'H1': [0, 137, 212],
	'H': [148, 205, 235],
	'T': [251, 251, 251],
	'P': [229, 234, 241],
	'F': [230, 255, 255]
};
```
Are they better? I do not know, we would have to ask some more people.

Reply:
I don't like the 'F' color where you just lowered the red value (starting off of white), which results in a blue-greenish look that is too greenish in my eyes.
![image](https://github.com/MaMpf-HD/mampf/assets/37160523/829a731e-3463-4710-b6ac-d75f0a264f3c)

If the argument was that the grey is too prominent, we could just replace it with a more light grey, so that the blue tiles attract more attention:

```js
const colors = {
	'H1': [0, 137, 212],
	'H': [148, 205, 235],
	'T': [251, 251, 251],
	'P': [250, 250, 250],
	'F': [220, 220, 220]
};
```

It looks like this which I think is a workable solution:

![image](https://github.com/MaMpf-HD/mampf/assets/37160523/d442ea52-d85d-4393-8e22-4cf91e9098ea)


Other than that, I thought one opinion is that the background shouldn't be too bright and not too colorful because MaMpf "is sober and should represent clarity and focus" [you or Hendrik Kasten]?. That's why I switched to the grey in the first place to have more subdued colors (even though I'd love to see the red color, but I agree that it is too much).

What were the complaints about the grey? This color is even is built-into the Mathematikon as facing concrete everywhere, so it perfectly represents the mathematical faculty ;) Of course, color choice is very subjective in the end.

Reply:
I like the colors of the previous comment, will approve if you use them.